docs/prepare-your-dev-environment.md

```diff
@@ -45,6 +45,21 @@ This document goes through the development dependencies one requires in order to
             podman machine init --cpus 4 --memory 5000
             podman machine start
             ```
+1.  Setting Up Podman with [Lima](https://github.com/ReToCode/local-cluster-setup/blob/main/lima/README.md) on macOS to to Resolve JSON Parse Errors when using 4.x clients.
```

Comment:
Can you please also point here that this is a temporary workaround, i.e. until an RHCOS image with Podman server 5.1.x becomes available?
Also,
Super Nit: ` on macOS to to Resolve` -> ` on macOS to Resolve`

Reply:
Done